emulated_hue/controllers/models.py

```diff
@@ -31,15 +31,15 @@ class EntityState(BaseModel):
 
     power_state: bool = True
     reachable: bool = True
-    transition_seconds: float | None = None
-    brightness: int | None = None
-    color_temp: int | None = None
-    hue_saturation: tuple[int, int] | None = None
-    xy_color: tuple[float, float] | None = None
-    rgb_color: tuple[int, int, int] | None = None
-    flash_state: str | None = None
-    effect: str | None = None
-    color_mode: str | None = None
+    transition_seconds: Optional[float] = None
```

Comment:
Can you revert these type changes? These are the old style python type hints and shouldn't be used. More over, these changes should be in a separate PR.

Reply:
i was able to fix the upstream issue by changing the tuple type of saturation to float instead of int